### ISSUE TITLE
Vala Errors

### ISSUE THREAD

**Issue by s5bug** [RedHatter/finalterm-reborn#5]
After cloning my own repo with the fixed semicolon, I get this:

```
$ make
[  3%] Generating src/FinalTerm.c, src/Terminal.c, src/TerminalStream.c, src/TerminalOutput.c, src/CharacterAttributes.c, src/TerminalView.c, src/TerminalWidget.c, src/NestingContainer.c, src/Autocompletion.c, src/NotifyingList.c, src/ScrollableTreeView.c, src/Utilities.c, src/SettingsWindow.c, src/TextMenu.c, src/ColorScheme.c, src/Theme.c, src/KeyBindings.c, src/Command.c, src/Settings.c, src/Metrics.c, src/StatusBar.c
/home/moo/finalterm-reborn/src/Settings.vala:293.3-293.40: warning: unhandled error `GLib.Error'
                style.load_from_data (css, css.length);
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
/home/moo/finalterm-reborn/src/TerminalOutput.vala:870.14-870.24: warning: unhandled error `GLib.RegexError'
                                } while (info.next());
                                         ^^^^^^^^^^^
/home/moo/finalterm-reborn/src/Command.vala:110.28-110.87: warning: unhandled error `GLib.RegexError'
                                substitute_parameter = placeholder_pattern.replace(substitute_parameter, -1, 0, "");
                                                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
/home/moo/finalterm-reborn/src/TerminalOutput.vala:650.35-650.37: error: The name `end' does not exist in the context of `TerminalOutput.interpret_stream_element'
                                                get_iter_at_line_offset(out end, get_line_length(i));
                                                                            ^^^
/home/moo/finalterm-reborn/src/TerminalOutput.vala:651.33-651.35: error: The name `end' does not exist in the context of `TerminalOutput.interpret_stream_element'
                                                this.delete(ref iter, ref end);
                                                                          ^^^
/home/moo/finalterm-reborn/src/Terminal.vala:64.12-64.57: warning: unhandled error `GLib.FileError'
                                return FileUtils.read_link(@"/proc/$(entry.key)/cwd");
                                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
/home/moo/finalterm-reborn/src/Terminal.vala:250.6-250.50: warning: unhandled error `GLib.IOChannelError'
                                        this_terminal.command_channel.shutdown(false);
                                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
/home/moo/finalterm-reborn/src/Terminal.vala:71.3-71.83: warning: unhandled error `GLib.SpawnError'
                Process.spawn_command_line_sync ("sh -c 'cat /proc/*/stat'", out stat, out error);
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
/home/moo/finalterm-reborn/src/Terminal.vala:91.3-91.83: warning: unhandled error `GLib.SpawnError'
                Process.spawn_command_line_sync ("sh -c 'cat /proc/*/stat'", out stat, out error);
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
/home/moo/finalterm-reborn/src/SettingsWindow.vala:29.15-29.23: warning: Gtk.Stock has been deprecated since 3.10
/home/moo/finalterm-reborn/src/NestingContainer.vala:392.4-400.13: warning: unhandled error `GLib.Error'
Compilation failed: 2 error(s), 9 warning(s)
CMakeFiles/finalterm.dir/build.make:115: recipe for target 'vala.stamp' failed
make[2]: *** [vala.stamp] Error 1
CMakeFiles/Makefile2:60: recipe for target 'CMakeFiles/finalterm.dir/all' failed
make[1]: *** [CMakeFiles/finalterm.dir/all] Error 2
Makefile:117: recipe for target 'all' failed
make: *** [all] Error 2
```

**Comment by yaleh** [comment]
Me too. Exactly the same error.

**Comment by Alcheorg** [comment]
Something like that:

`make

[  3%] Generating src/FinalTerm.c, src/Terminal.c, src/TerminalStream.c, src/TerminalOutput.c, src/CharacterAttributes.c, src/TerminalView.c, src/TerminalWidget.c, src/NestingContainer.c, src/Autocompletion.c, src/NotifyingList.c, src/ScrollableTreeView.c, src/Utilities.c, src/SettingsWindow.c, src/TextMenu.c, src/ColorScheme.c, src/Theme.c, src/KeyBindings.c, src/Command.c, src/Settings.c, src/Metrics.c, src/StatusBar.c
/home/**_/Programs/finalterm/finalterm-reborn/src/TerminalOutput.vala:651.7-651.10: error: syntax error, expected `;'
                        this.delete(ref iter, ref end);
                        ^^^^
/home/**_/Programs/finalterm/finalterm-reborn/src/TerminalOutput.vala:651.23-651.26: error: syntax error, expected `)'
                        this.delete(ref iter, ref end);
                                        ^^^^
Compilation failed: 2 error(s), 0 warning(s)
CMakeFiles/finalterm.dir/build.make:144: recipe for target 'vala.stamp' failed
make[2]: *** [vala.stamp] Error 1
CMakeFiles/Makefile2:67: recipe for target 'CMakeFiles/finalterm.dir/all' failed
make[1]: *** [CMakeFiles/finalterm.dir/all] Error 2
Makefile:127: recipe for target 'all' failed
make: *** [all] Error 2
`

**Comment by OctoNezd** [comment]
Same thing

**Comment by SitanHuang** [comment]
Same thing~

**Comment by RedHatter** [comment]
Fixed in 312a65e08c90f7278594a89aaa74df2b49d38874

**Comment by OctoNezd** [comment]
THE PROJECT IS ALIVE!
